Lib/fontbakery/profiles/name.py

```diff
@@ -482,7 +482,8 @@ def com_adobe_fonts_check_name_postscript_vs_cff(ttFont):
     """CFF table FontName must match name table ID 6 (PostScript name)."""
     cff_names = ttFont['CFF '].cff.fontNames
     if len(cff_names) != 1:
-        yield ERROR, "Unexpected number of font names in CFF table."
+        yield ERROR, Message("cff-name-error",
```

Comment:
hey! This looks like it should have always been a FAIL instead of an ERROR, right?

Reply:
Yeah, I think you're right.

Reply:
perfect, thanks!

Reply:
Changed in 6df7466aea3b8fb653baefd42941a7373eb86b5f